portfolio/src/main/java/com/google/sps/servlets/Testimonials.java

```diff
@@ -0,0 +1,21 @@
+package com.google.sps.servlets; // is this okay?
+
+import java.util.ArrayList;
+
+public class Testimonials { 
+   private ArrayList<String> arrTestimonials; 
+
+   public Testimonials(){} 
+   
+   public ArrayList<String> getArrTestimonials() { 
+      return arrTestimonials; 
+   }
+   
+   public void setArrTestimonials(ArrayList<String> arrTestimonials) { 
+      this.arrTestimonials = arrTestimonials; 
+   } 
+   
+   public String toString() { 
+      return "Testimonials [ arrTestimonials: "+arrTestimonials+",]"; 
```

Comment:
Does this print out every testimonial? If not, consider building the string by iterating over each testimonial and concatenating.

Reply:
Yes it should print every testimonial because of the arrayList toString method